cerberus/validator.py

```diff
@@ -899,10 +899,6 @@ def _validate_dependencies(self, dependencies, field, value):
         elif isinstance(dependencies, Mapping):
             self.__validate_dependencies_mapping(dependencies, field)
 
-        if self.document_error_tree.fetch_node_from(
```

Comment:
I didn't fully understand what this code did, but nobody was looking at the result until now, so I removed it in order to avoid aborting the validation check.